client/CMakeLists.txt

```diff
@@ -177,9 +177,16 @@ endif()
 
 target_link_libraries(vcmiclient PRIVATE
 		vcmi SDL2::SDL2 SDL2::Image SDL2::Mixer SDL2::TTF
-		ffmpeg::swscale ffmpeg::avutil ffmpeg::avcodec ffmpeg::avformat
 )
 
+if(ffmpeg_LIBRARIES)
+	target_link_libraries(vcmiclient PRIVATE
+		ffmpeg::swscale ffmpeg::avutil ffmpeg::avcodec ffmpeg::avformat
+	)
+else()
+	target_compile_definitions(vcmiclient PRIVATE DISABLE_VIDEO)
```

Comment:
Really we even have this flag DISABLE_VIDEO in our code base.

Reply:
Lets try to visit tavern just in case

Reply:
I have tested the following cases with video code disabled:
- intro
- a few campaigns
- tavern 

Also tested removing .vid files with the code enabled - works fine in the 1.0.0 release.